### ISSUE TITLE
No SD Card Issue!

### ISSUE THREAD

**Issue by PercyRosasR** [ClusterM/fdskey#13]
Hello Alexey and everyone.

I am having the next issue with my unit.

When I turn my fdskey without a micro sd card I got this message.
![IMG_20231014_083810](https://github.com/ClusterM/fdskey/assets/135164149/9f403993-56f3-4fe5-8e0f-cb5bd47d71a4)

When I insert the micro sd, the screen turns black without response. It looks like it's turned off or something different happened.

![IMG_20231014_083828](https://github.com/ClusterM/fdskey/assets/135164149/36c3ade2-928c-43d8-93ca-d6ed9270af95)

If I start the console with the micro sd inserted, the same problem shows, a black screen.

I have already tried with 2 micro sd, one 2gb Samsung, and 16gb sandisk.

I am adding a YouTube video of the problem.

https://youtube.com/shorts/ha5OHcCc-NU?feature=share

-------------------------------------------------------------------------------------------

Also I tried to update by pressing the four buttons but it get stuck with the "Please Wait" message:

![please wait](https://github.com/ClusterM/fdskey/assets/135164149/d695f4e2-b1c6-4ed7-9874-8051c2b3b96c)

I hope you can help me solve this issue.

*I had reweld the sd card holder pins and got the same issue.

Thanks in advance.

**Comment by ClusterM** [comment]
Seems like the card init process has not even started. Please check the card detect line/pin. Sometimes the card detect mechanism can also be broken. Try to short the card detect line to the ground. The card socket must connect it to the ground when an SD card is inserted.

**Comment by PercyRosasR** [comment]
Hi Alexey, good morning and greetings from Peru.

I shorted the cd line to the ground and I got a black screen.

I have already made 3 of the fdskey boards to test and all those have the same behavior.

What else could it be? Do you think it might be faulty micro sd card sockets?

Let me know if you need more detail, I followed all the instructions, but I don't know if could be a bad capacitor or resistor.

Thanks in advance for your help.

**Comment by ClusterM** [comment]
Weird. I have no idea.

**Comment by PercyRosasR** [comment]
After doing some additional tests, I was checking the voltages in the board.

And I was having less than 3.3v, in some cases 2.6v. Then I realized that I had a mod to connect USB C to give power to the console.

So I connected the regular 10v adapter and my unit came alive, so was a mistake from my last mode.
![working](https://github.com/ClusterM/fdskey/assets/135164149/0252ed80-a0d3-4a35-bfc1-1905365de53e)

I will try to fix that, but the system is pretty good and works properly.

Thanks, Alexey for your help.

This device is pretty awesome.
